simulation/traffic_simulator/src/entity/entity_base.cpp

```diff
@@ -217,6 +217,11 @@ auto EntityBase::getLaneletPose(double matching_distance) const
   }
 }
 
+auto EntityBase::fillLaneletPose(traffic_simulator_msgs::msg::EntityStatus & status) const -> void
+{
+  THROW_SIMULATION_ERROR(status.name, " entity does not support fillLaneletPose.");
```

Comment:
All entity can fill lanelet pose, so throwing errors is not desirable.
Please change `EntityBase::fillLaneletPose` function to the pure virtual function.